jsonrpc.go

```diff
@@ -451,11 +451,16 @@ func (rpcResponse *RPCResponse) GetObject(toType interface{}) error {
 		return err
 	}
 
-	err = json.Unmarshal(js, toType)
+	err = json.Unmarshal(js, &toType)
```

Comment:
why did you add the operator here? isnt interface{} always a pointer?

Reply:
yes, when the result is "null", `toType` can be `nil`, rather than a `struct` or something.

Reply:
please add a test that is only green with this fix.

Reply:
@niulingyun 
Sorry for the delay, I see what's your point, and you are correct.

I will take a closer look e.g. under which circumstances this fix would break client code.

But it seems reasonable to merge this PR